src/containers/Tenant/Diagnostics/Consumers/Consumers.tsx

```diff
@@ -59,8 +77,20 @@ export const Consumers = ({path}: ConsumersProps) => {
         },
     ];
 
+    if (loading && !wasLoaded) {
+        return (
+            <div className={b('loader')}>
+                <Loader size="m" />
+            </div>
+        );
+    }
+
+    if (!loading && error) {
```

Comment:
case when `!loading && error` should not be real. Please check your reducer to make sure that error is set to undefined when start a new request.

Reply:
Actually, it is set to undefined on request success. Isn't that enough?

Reply:
Discussed in PM, this is done to avoid error text blinking when requests are failing in a row when autofetcher is enabled.